doc/doxygen/CMakeLists.txt

```diff
@@ -158,11 +158,13 @@ file(GLOB _doxygen_depend
 # are captured via a GLOB since they may not exist (and so not
 # be captured via the GLOB) at the time cmake runs
 LIST(APPEND _doxygen_depend
-  ${CMAKE_BINARY_DIR}/doc/news/changes.h
   ${CMAKE_BINARY_DIR}/include/deal.II/base/config.h
   ${CMAKE_CURRENT_BINARY_DIR}/tutorial/tutorial.h
   )
 
+# Add the changelog (a CMake custom target) as a dependency for doxygen too
+LIST(APPEND _doxygen_depend changelog)
```

Comment:
Ha, interesting viewpoint. To me targets are rules to generate some files from other files. Outputs and inputs are files, and dependencies consequently are files as well. That's because the input file needs to exist, but a dependency target does not need to run if its output already exists.

But that's really all not very important :-)